test/doc/test_common.py

```diff
@@ -46,15 +45,15 @@
     #> 2+2
      = 4
     A private doctest with a message
-    #> 1/0 
-     : 
+    #> 1/0
+     :
```

Comment:
The test in line 89 fails because this line should be
`: Infinite expression 1 / 0 encountered.`